includes/wikia/resourceloader/ResourceLoaderGlobalWikiModule.class.php

```diff
@@ -99,22 +102,22 @@ protected function getContent( $title, $titleText, $options = array() ) {
 			*/
 
 			if ( WikiFactory::isWikiPrivate( $title->getCityId() ) == false ) {
-				$content = $title->getContent();
+				if ( !is_null( $revisionId ) ) {
```

Comment:
I think that https://github.com/Wikia/app/commit/075420341e6342c21cf02176461f87594dd56615 fixes it :)